### ISSUE TITLE
Monitoring: Applied restrictions are taking effect only after the user logs in again

### ISSUE THREAD

**Issue by Yonas-net** [Icinga/icingaweb2#4420]
## Describe the bug
I don't know 🤔 for sure if this has always been the case, but if you filter in the restrictions for certain objects, you don't see the effect until you re-login. It also doesn't help if you hard reload the browser umpteen times.

## To Reproduce

Navigate to ``/icingaweb2/role/list#!/icingaweb2/role/edit?role=rolename`` and in ``monitoring/filter/objects`` add some restrictions and save the role. And if you go back to Dashboard now, the recently saved restriction has no effect at all, even if you hard reload the browser.

**Comment by nilmerg** [comment]
Hey, yeah, that's been always the case and will be tackled together with #3162

**Comment by nilmerg** [comment]
Duplicate of #3555.
